Code/Uplynk/Source/Internal/UplynkPingFeatures.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  UplynkPingFeatures.swift
+//  THEOplayer-Connector-Uplynk
+//
+//  Created by Khalid, Yousif on 28/1/2025.
+//
+
+import Foundation
+
+enum UplynkPingFeatures: Int {
+    case noPing
+    case adImpressions
+    case fwVideoViews
+    case adImpressionsAndFwVideoViews
+    case linearAdData
+
+    
+    init(ssaiConfiguration: UplynkServerSideAdIntegrationConfiguration) {
+        let isVod = ssaiConfiguration.assetType == .asset
+        let pingConfiguration = ssaiConfiguration.pingConfiguration
+        switch (isVod,
+                pingConfiguration.adImpressions,
+                pingConfiguration.freeWheelVideoViews,
+                pingConfiguration.linearAdData) {
+        case (true, true, true, _):
+            self = .adImpressionsAndFwVideoViews
+        case (true, true, _, _):
```

Comment:
Trivial: May be `(true, true, _, _)` -> `(true, true, false, _)` improves readability?